todo/views.py

```diff
@@ -0,0 +1,75 @@
+from django.shortcuts import get_object_or_404, redirect
+from django.urls import reverse_lazy
+from django.views import generic, View
+
+from todo.forms import TagForm, TaskForm
+from todo.models import Task, Tag
+
+
+class TaskListView(generic.ListView):
+    model = Task
+    context_object_name = "task_list"
+    template_name = "todo/task_list.html"
+    paginate_by = 3
+    queryset = Task.objects.prefetch_related("tags")
```

Comment:
Why don't you used reverse_lazy method to refer to the correct URL, as you did it in TaksUpdateView, or in TagDeleteView classes? 
This success_url attribute of the TaskCreateView and TagCreateView classes can be changed to use reverse_lazy and refer to the correct URL